src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -46,61 +46,90 @@ export type ImportAccountsProps = {
 }
 
 type TableRowProps = {
-  accountId: AccountId
+  accountIds: AccountId[]
   accountNumber: number
   asset: Asset
-  onAccountIdActiveChange: (accountId: AccountId, isActive: boolean) => void
+  onAccountIdsActiveChange: (accountIds: AccountId[], isActive: boolean) => void
+}
+
+type TableRowAccountProps = {
+  accountId: AccountId
+  asset: Asset
 }
 
 const disabledProps = { opacity: 0.5, cursor: 'not-allowed', userSelect: 'none' }
 
+const TableRowAccount = forwardRef<TableRowAccountProps, 'div'>(({ asset, accountId }, ref) => {
+  const translate = useTranslate()
+  const accountLabel = useMemo(() => accountIdToLabel(accountId), [accountId])
+  const pubkey = useMemo(() => fromAccountId(accountId).account, [accountId])
+  const isUtxoAccount = useMemo(() => isUtxoAccountId(accountId), [accountId])
+
+  const { data: account, isLoading } = useQuery(accountManagement.getAccount(accountId))
+
+  const assetBalancePrecision = useMemo(() => {
+    if (!account) return '0'
+    return fromBaseUnit(account.balance, asset.precision)
+  }, [account, asset.precision])
+
+  return (
+    <>
+      <Td fontWeight='bold'>
+        <Tooltip label={pubkey} isDisabled={isUtxoAccount}>
+          <div ref={ref}>
+            {isUtxoAccount ? (
+              <RawText>{`${accountLabel} ${translate('common.account')}`}</RawText>
+            ) : (
+              <MiddleEllipsis value={accountLabel} />
```

Comment:
I think this looks kinda wack for some chains like Cosmos and Thorchain, but it might be a spec issue and not a PR issue:

<img width="452" alt="Screenshot 2024-05-06 at 2 09 45 PM" src="https://github.com/shapeshift/web/assets/97164662/afea9c4b-6753-4cc7-9722-1e5fdb54c027">

<img width="450" alt="Screenshot 2024-05-06 at 2 10 17 PM" src="https://github.com/shapeshift/web/assets/97164662/3f890a33-c6b1-4b61-a459-49ac5bce0637">

Reply:
That's definitely a regression - we had spotted this one previously for UTXOs.
@woodenfurniture since THORChain and Cosmos are account-based, we should definitely be able to use addresses as pubkeys here

Reply:
Sounds good, I'll make a ticket to follow up so we can close the 2 approved downstream PRs

Reply:
Thorch..ch...ch..hain

Reply:
https://github.com/shapeshift/web/issues/6837